amico/core.py

```diff
@@ -59,6 +59,7 @@ def __init__( self, study_path, subject, output_path=None ) :
         self.set_config('doKeepb0Intact', False)  # does change b0 images in the predicted signal
         self.set_config('doComputeNRMSE', False)
         self.set_config('doSaveCorrectedDWI', False)
+        self.set_config('doMergeB0', True) # Merge b0 volumes
```

Comment:
Adressed in https://github.com/davidrs06/AMICO/blob/05417a3688c6e276e489948333792bc58a3ac72d/amico/core.py#L62